README.md

```diff
@@ -1,2 +1,89 @@
 # CapstoneJS-API-Webapp
 This is a project that involves the creation of a Webapp consumming an API
+
+
+# CapstoneJS-API-Webapp
+
+> This is a project that involves the creation of a Webapp consumming two APIs services.
+
+![Desktop_image](./desktop_ver.png) 
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML5
+- CSS3
+- JavaScript
+- Bootstrap
+- Linters
+- API: Pokemon TCG
+- API: Involvement Microverse
+
+If you like, you can see a live demo of the site here:
+
+[Live Demo Link](https://angelacuahutle.github.io/CapstoneJS-API-Webapp/dist)
```

Comment:
- [ ] Since your partner isn't available kindly consider deploying the project on Heroku or Netlify and update the live deploy link in the readme.

Reply:
Thanks for your feedback @mikenath223, With all the respect that you deserve, but you ask me to investigate a new way of deploying this page, and since in microverse only taught us to do this on GitHub, given the short time that we are confronting right now, could you please at least give some instructions about how to do this exactly, and not just the "first answer" that google returns. I don't use Heroku or Netlify.

Reply:
OK, Instead of the idea that you were giving me about deploying on unknown platforms that Microverse still not give us even lessons to learn about, I use the tool that Microverse teaches us to use: GITHUB. I forked the repo and deploy it with my user. The Change is made in commit 866e028.